src/pages/home/report/EmojiSkinToneList.js

```diff
@@ -0,0 +1,108 @@
+    preferredSkinTone: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+
+    /** Function to sync the selected skin tone with parent, onyx and nvp */
+    setPreferredSkinTone: PropTypes.func.isRequired,
+
+    /** Size of the emoji item */
+    emojiSize: PropTypes.shape({
+        fontSize: PropTypes.number,
+    }).isRequired,
+
+    toggleSkinToneList: PropTypes.func.isRequired,
+
+
+    /** Props related to translation */
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    isSkinToneListVisible: false,
+};
+
+const EmojiSkinToneList = (props) => {
+    const selectedEmoji = getSkinToneEmojiFromIndex(props.preferredSkinTone);
+    const [highlightedIndex, setHighlightedIndex] = useState(selectedEmoji.skinTone);
+    return (
+        <View style={[styles.flexRow, styles.p1]}>
+            {
+            !props.isSkinToneListVisible
+                && (
+                <Pressable
+                    onPress={
+                      () => props.toggleSkinToneList(!props.isSkinToneListVisible)
+                    }
+                    style={[
+                        styles.pv1,
+                        styles.flex1,
+                        styles.flexRow,
+                        styles.alignSelfCenter,
+                        styles.justifyContentStart,
+                    ]}
+                >
+                    <Text style={[styles.emojiText, props.emojiSize]}>
+                        {`${selectedEmoji.code}\uFE0F`}
+                    </Text>
+                    <Text style={[styles.emojiHeaderStyle]}>
+                        {props.translate('emojiPicker.skinTonePickerLabel')}
+                    </Text>
+                </Pressable>
+                )
+        }
+            {
+            props.isSkinToneListVisible
+                && (
+                <View>
+                    <View style={[styles.flexRow]}>
+                        {
+                      skinTones.map(skinToneEmoji => (
+                          <EmojiPickerMenuItem
+                              onPress={() => props.setPreferredSkinTone(skinToneEmoji.skinTone)}
+                              onHover={() => setHighlightedIndex(skinToneEmoji.skinTone)}
+                              key={skinToneEmoji.code}
+                              emojiItemStyle={[styles.emojiSkinToneItem, styles.emojiSkinToneItem]}
+                              emoji={`${skinToneEmoji.code}\uFE0F`}
+                              isHighlighted={skinToneEmoji.skinTone === highlightedIndex}
+                              emojiSize={props.emojiSize}
+                          />
+                      ))
+                    }
+                    </View>
+                    <Text style={
+                      [styles.emojiHeaderStyle, styles.emojiExtraSmall, styles.textAlignCenter, styles.pv0]
+                      }
+                    >
+                        {props.translate('emojiPicker.setPreferredSkinTone')}
```

Comment:
I don't think we need this, it's pretty self explanatory IMO

Reply:
Removed